.eslintrc.yml

```diff
@@ -1,9 +1,10 @@
 ---
 extends: eslint-config-shakacode/base
+rules:
+  import/no-unresolved: off
```

Comment:
Why was this change necessary?

Reply:
couldn't make ESLint work for example without ignoring that rule. I tried many settings and resolvers such as alias, node and webpack in .eslintrc.yml. An error Unable to resolve path to module 'react' and all other imported modules kept popping on CI, while locally there were no such issues. So I decided that expanding ESLint to check other errors is more important than to keep dealing with this one. I hope it's a temporary workaround, that should be fixed in one of the following PRs.